<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_interface_menu</class>
 <widget class="QWidget" name="page_interface_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Context Menu Plugins</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="list_menu"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QToolButton" name="tool_menu_add">
       <property name="text">
        <string notr="true">add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tool_menu_rm">
       <property name="text">
        <string notr="true">rem</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="tool_menu_up">
       <property name="text">
        <string notr="true">up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tool_menu_dn">
       <property name="text">
        <string notr="true">dn</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
